index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <script src="https://kit.fontawesome.com/8a30dfc1c1.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="style.css">
+
+    </head>
+
+      <div class="headline-wrapper">
+        <div class="app-wrap">
+        
+          <header class="app-header">
+            <a href="#" class="button">
+              <i class="fas fa-bars"></i>
+            </a>
+            
+            <h1>My Logo</h1>  
+        
+          </header>
+
+         <div class="content">
+           <h2>I'm Vana<br> Glad to see<br> you!</h2>
```

Comment:
- [ ] Please kindly replace `h2` tag with `h1` tag, because this is the title of your document 👍🏻